test/integration/developer_portal/base_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module DeveloperPortal
+  class BaseControllerTest < ActionDispatch::IntegrationTest
+
+    class FilterReadOnlyParams < BaseControllerTest
```

Comment:
```suggestion
    class FilterReadOnlyParamsTest < BaseControllerTest
```
Also indentation appears to be broken.

Reply:
What's the problem with indentation? it looks right to me

Reply:
ok, I didn't see it was nesting